code/game/objects/structures/watercloset.dm

```diff
@@ -515,6 +515,19 @@
 			to_chat(user, SPAN_NOTICE("You empty the [O] into the [src]."))
 
 
+/obj/structure/sink/fountain
+	name = "water fountain"
+	desc = "A small fountain of polished stone and has a small pump at the bottom that helps push fresh water."
```

Comment:
"and" should probably be "that" here as well

Reply:
```suggestion
	desc = "A small fountain of polished stone that has a small pump at the bottom which helps pump out fresh water."
```